packages/zengine-ui-react/src/util/FocusFormErrors.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
+import { useFormikContext } from 'formik';
+
+const FocusFormErrors = (): null => {
+  const { errors, isSubmitting, isValidating } = useFormikContext();
+
+  useEffect(() => {
+    if (isSubmitting && !isValidating) {
+      const keys = Object.keys(errors);
+      if (keys.length > 0) {
+        const selector = `[name=${keys[0]}]`;
+        const errorElement = document.querySelector(selector) as HTMLElement;
+        if (errorElement !== null) {
+          window.scrollTo({
```

Comment:
this is problematic for IE. Better to use css `* { scroll-behavior: smooth }` and just use the x, y arguments for `scrollTo`.

https://developer.mozilla.org/en-US/docs/Web/API/ScrollToOptions

example:
https://github.com/Wizehive/zn-plugin-reporting-dashboard/blob/bc48bdfaaee75195147ce54545f2f3a9920ab8a7/plugins/dashboard/src/utilities/index.js#L148
https://github.com/Wizehive/zn-plugin-reporting-dashboard/blob/bc48bdfaaee75195147ce54545f2f3a9920ab8a7/plugins/dashboard/src/dashboard.css#L3

Reply:
Good catch, I'll update this...